packages/blade/src/components/Icons/ArrowDownRightIcon/ArrowDownRightIcon.tsx

```diff
@@ -1,8 +1,13 @@
 import { Svg, Path } from '../_Svg';
 import type { IconComponent } from '..';
 import useIconProps from '../useIconProps';
+import { defaultProps } from '../defaults';
 
-const ArrowDownRightIcon: IconComponent = ({ size, color, ...styledProps }) => {
+const ArrowDownRightIcon: IconComponent = ({
+  size = defaultProps.size,
+  color = defaultProps.color,
+  ...styledProps
+}) => {
```

Comment:
fyi before you try to change this on all 260 icons, 

It might be better to change the defaults inside the useIconProps hook. 

> though we can also change it on all 260 icons individually, we actually have a script to autogenerate icons.

Reply:
I think it will be better to change this for each icon individually so that the user can use the components right away without providing any props. Can you please tell me about the script?